pxr/usdImaging/usdImaging/materialAdapter.cpp

```diff
@@ -379,6 +379,19 @@ UsdImagingMaterialAdapter::GetMaterialResource(UsdPrim const &prim,
             time);
     }
 
+    // finally collect any 'info' on the Material prim
+    VtDictionary infoDict;
```

Comment:
(1) Do we only want to include authored attributes, or are we expecting the fallback of "1.38" to ever be useful?  This won't pull fallbacks, but GetAttributes would().  Maybe moot since usdMtlx/reader.cpp looks to be manually creating the attribute.

(2) We may want to use prim.GetPropertiesInNamespace("info") (or GetAuthoredPropertiesInNamespace) instead here?  And then auto attr = prop.As<UsdAttribute>(), which returns null if it's not an attribute (e.g. if it's a relationship).  See also: https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usd/usdGeom/primvarsAPI.cpp#L215.

Reply:
I can't see that it would hurt any.